WCPT: Move post type check to before nonce check.

vedanshujain opened the pull request:
This was giving an error sometimes while submitting a post, looks like because `metabox_save` is getting enqueued some other place as well.

iandunn commented on public_html/wp-content/plugins/wcpt/wcpt-event/class-event-admin.php:
```diff
@@ -412,7 +412,11 @@ public function metabox_save( $post_id, $post ) {
 		}
 
-		// Make sure the requset came from the edit post screen.
+		if ( $this->get_event_type() !== get_post_type() ) {
```
Using `$post` might be a bit safer than `get_post_type()`. I can't think of a case where they'd be different, but there might be some.

vedanshujain committed:
WCPT: Move post type check to before nonce check.

This was giving an error sometimes while submitting a post, looks like because `metabox_save` is getting enqueued some other place as well.

git-svn-id: https://meta.svn.wordpress.org/sites/trunk/wordcamp.org@8133 74240141-8908-4e6f-9713-ba540dce6ec7
